content/efcore/getting-started/samples/defining-unique-identifiers/composite-key/LibraryContext.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.EntityFrameworkCore;
+using REPLHelper;
+
+public class LibraryContext : DbContext
+{
+    public DbSet<CheckoutRecord> CheckoutRecords { get; set; }
```

Comment:
Could you add Books and Readers? EF Core detects them automatically, but I would still like them here please.